src/utils/reallocationMaths.ts

```diff
@@ -122,9 +116,19 @@ export const getMarketReallocationData = (
 
 export const seekForSupplyReallocation = (
   marketToSupplyIntoId: string,
-  vault: MetaMorphoVault
+  vault: MetaMorphoVault,
+  filterIdleMarkets: boolean
 ): Reallocation | undefined => {
-  console.log("seeking for supply reallocation for vault:", vault.link.name);
+  if (
+    filterIdleMarkets &&
+    vault.positions[marketToSupplyIntoId].marketData.collateralAsset.address ===
+      "0x0000000000000000000000000000000000000000"
+  ) {
+    console.log(
+      "collateral asset is undefined, this corresponds to an idle market we don't want to reallocate in it"
+    );
+    return undefined;
+  }
```

Comment:
this is actually useless because as their is now targets for idle markets, they will never be considered "out of range"

Therefore this `marketToSupplyIntoId` will never be an idle market